src/EditorFeatures/Test/Utilities/PatternMatcherTests.cs

```diff
@@ -195,67 +197,54 @@ private void VerifyBreakIntoCharacterParts(string original, params string[] part
         [InlineData("_my[|_B|]utton", "_b", PatternMatchKind.Substring, CaseInsensitive)]
         [InlineData("_[|myB|]utton", "myb", PatternMatchKind.Substring, CaseInsensitive)]
 
-        [InlineData("[|AbCd|]xxx[|Ef|]Cd[|Gh|]", "AbCdEfGh", PatternMatchKind.CamelCase, CaseSensitive, PatternMatcher.CamelCaseMatchesFromStartBonus)]
+        [InlineData("[|AbCd|]xxx[|Ef|]Cd[|Gh|]", "AbCdEfGh", PatternMatchKind.CamelCaseFromStart, CaseSensitive)]
```

Comment:
CameCaseWeights are gone.  They're now represented just as the 'Kind' of hte match.